contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/LicenseStatus.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+
+  <Box
+    :kind="boxKind"
+    :loading="isLoading"
+    data-test="license-status"
+  >
+    <template #title>
+      {{ titleText }}
+    </template>
+    <template #description>
+      {{ descriptionText }}
+    </template>
+  </Box>
+
+</template>
+
+
+<script>
+
+  import { computed } from 'vue';
+  import Box from './Box.vue';
+  import { useLicenseNames } from './composables/useLicenseNames';
+  import { communityChannelsStrings } from 'shared/strings/communityChannelsStrings';
+
+  const {
+    licenseCheckPassed$,
+    allLicensesCompatible$,
+    incompatibleLicensesDetected$,
+    channelCannotBeDistributed$,
+    fixLicensingBeforeSubmission$,
+  } = communityChannelsStrings;
+
+  export default {
+    name: 'LicenseStatus',
+    components: {
+      Box,
+    },
+    setup(props) {
+      const invalidLicenseIds = computed(() => {
+        if (!props.invalidLicenses || props.invalidLicenses.length === 0) {
+          return [];
+        }
+        return props.invalidLicenses;
+      });
+
+      const includedLicenseIds = computed(() => {
+        if (!props.includedLicenses || props.includedLicenses.length === 0) {
+          return [];
+        }
+        return props.includedLicenses;
+      });
+
+      const { formattedLicenseNames: invalidLicenseNames, isLoading: isLoadingInvalid } =
+        useLicenseNames(invalidLicenseIds);
+
+      const { formattedLicenseNames: includedLicenseNames, isLoading: isLoadingIncluded } =
```

Comment:
Looking at the Figma designs, it seems that we should not be displaying the "Special permissions" as an included license in the license check passed notice [here](https://www.figma.com/design/0R2sZFZQJd9iPiOJ2LzVTR/Easy-sharing-of-community-channels?node-id=1382-9398&t=ZpICKr0BAwJIiRk3-0). So, just an additional request -- could you please filter out this license before formatting the license names? Thanks!